index.html

```diff
@@ -0,0 +1,350 @@
+          <figcaption>
+            <h4 class="portfolio-caption-header">Korean beef</h4>
+            <p class="portfolio-category">Webdesign, brand &amp; logotypes</p>
+          </figcaption>
+        </figure>
+
+        <figure class="portfolio-item">
+          <a class="portfolio-link-hover" href=""><img class="portfolio-opacity" src="/img/facebook-thumb.jpg" alt="Image of Facebook like and dislike thumb icons" title="Facebook thumbs"></a>
+          <figcaption>
+            <h4 class="portfolio-caption-header">Facebook thumbs</h4>
+            <p class="portfolio-category">Icons design, brand &amp; logotypes</p>
+          </figcaption>
+        </figure>
+
+      </article>
+      <article id="about" class="about">
+        <h2 class="about-header white-line">ABOUT</h2>
+        <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.</p>
+        <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.</p>
+        <section>
+          <h3 class="skills-header white-line">SKILLS</h3>
+          <ul class="skills">
+            <li>
+              <h4 class="progress-header">PHOTOSHOP</h4>
+              <progress value="77" max="100"></progress>
+            </li>
+            <li >
+              <h4 class="progress-header">HTML/CSS</h4>
+              <progress value="91" max="100"></progress>
+            </li>
+            <li>
+              <h4 class="progress-header">ILLUSTRATOR</h4>
+              <progress value="69" max="100"></progress>
+            </li>
+            <li>
+              <h4 class="progress-header">PHP</h4>
+              <progress value="55" max="100"></progress>
+            </li>
+            <li>
+              <h4 class="progress-header">WORDPRESS</h4>
+              <progress value="97.5" max="100"></progress>
+            </li>
+          </ul>
+        </section>
+        <aside>
+          <h3 class="join-us-header">Do you want to be one of Avengers? Join us!</h3>
+          <button class="contact-us-button">CONTACT US</button>
+        </aside>
+      </article>
+      <article id="team" class="team">
+        <h2 class="team-section-header gray-line">TEAM</h2>
+        <p class="team-text team-description-text">Contrary to popular belief, Lorem Ipsum is not simply random text. It has roots in a piece of classical Latin literature from 45 BC, making it over 2000 years old. Richard McClintock, a Latin professor at Hampden-Sydney College in Virginia.</p>
+        <section class="team-profile">
```

Comment:
Again, like the services section, I would mark this up as a list rather than a collection of `section`s. Semantically it makes more sense, as you're displaying a number of similar items (in this case, team members).